internal/ignition/ignition.go

```diff
@@ -288,15 +290,24 @@ func (g *installerGenerator) UploadToS3(ctx context.Context) error {
 
 // Generate generates ignition files and applies modifications.
 func (g *installerGenerator) Generate(ctx context.Context, installConfig []byte, platformType models.PlatformType) error {
+	var icspFile string
 	log := logutil.FromContext(ctx, g.log)
 
 	// In case we don't want to override image for extracting installer use release one
 	if g.installerReleaseImageOverride == "" {
 		g.installerReleaseImageOverride = g.releaseImage
 	}
 
+	// If ImageContentSources are defined, store in a file for the 'oc' command
+	defer removeIcspFile(icspFile)
```

Comment:
Arguments are evaluated at the time of the defer, so this will always pass `""`. Need to move below line 304.

Reply:
Good catch, fixed.